api_v3/services/PartnerService.php

```diff
@@ -139,7 +139,8 @@ public function registerAction( KalturaPartner $partner , $cmsPassword = "" , $t
 		{
 			KalturaLog::CRIT($ex);
 			// this assumes the partner name is unique - TODO - remove key from DB !
-			throw new KalturaAPIException( APIErrors::PARTNER_REGISTRATION_ERROR, 'Unknown error');
+			$exceptionMessage = (str_replace('KalturaPartner::', '', $ex->getMessage()));
```

Comment:
keep the original exception catch as it was and instead add a catch for kCoreException that expose the error massage

Reply:
done